apps/speed.c

```diff
@@ -2688,8 +2688,8 @@ int speed_main(int argc, char **argv)
                    k, rsa_bits[k], rsa_results[k][0], rsa_results[k][1]);
         else
             printf("rsa %4u bits %8.6fs %8.6fs %8.1f %8.1f\n",
```

Comment:
Now you have changed the maths round, do we still require 6 decimal places on these fields? I think the info in the decimal places will now get thrown away after the maths on the other end of the pipe when the data is formatted for output. This also applies to DSA, ECDH, ECDSA as well.